corehq/motech/repeaters/tests/test_models_slow.py

```diff
@@ -19,7 +19,7 @@
     RECORD_FAILURE_STATE,
     RECORD_SUCCESS_STATE,
 )
-from corehq.motech.repeaters.models import FormRepeater, send_request
+from corehq.motech.repeaters.models import FormRepeater
```

Comment:
We really ought to get better at this.

https://github.com/dimagi/commcare-hq/pull/34995